tools/dune_rule_gen/coq_rules.ml

```diff
@@ -0,0 +1,185 @@
+(************************************************************************)
+(* This file is licensed under The MIT License                          *)
+(* See LICENSE for more information                                     *)
+(* (c) MINES ParisTech 2018-2019                                        *)
+(* (c) INRIA 2020-2022                                                  *)
+(* Written by: Emilio Jesús Gallego Arias                               *)
+(* Written by: Rudi Grinberg                                            *)
+(************************************************************************)
+
+(* coq_rules: generate dune build rules for Coq's test-suite and stdlib *)
+
+(* Originally written by Emilio Jesús Gallego Arias and Rudi Grinberg
+   for Dune's coq_rules.ml , which followed a first coq_dune.exe
+   implementation by Emilio Jesús Gallego Arias *)
+
+(* note that this file closely follows Dune's own coq_rules.ml [and
+   eventually they could be merged], hence the license. Other modules
+   such as Arg or Path are straight copies from Dune's *)
+
+let _debug = false
+
+module FlagUtil = struct
+
+  let plugin_list =
+    [ "ltac"
```

Comment:
this list should really be dynamically generated

Reply:
Yes, or even better we can try to figure out how to have coqdep use the meta file and get rid of the list altogheter. Code is there, but I don't understand it yet.